src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -104,7 +99,11 @@ export class SmartSnippet extends Component {
   }
 
   private buildHeightLimiter() {
-    return (this.heightLimiter = new HeightLimiter(this.shadowContainer, 400)).toggleButton;
+    return (this.heightLimiter = new HeightLimiter(
+      this.shadowContainer,
+      400,
+      isExpanded => (isExpanded ? this.sendExpandSmartSnippetAnalytics() : this.sendCollapseSmartSnippetAnalytics())
```

Comment:
Should the analytics be reversed here? If expanded, then onclick we send a collapse analytics event?

Reply:
`isExpanded` represents the state after the button was clicked.